Source/Core/AudioCommon/CMakeLists.txt

```diff
@@ -71,6 +71,11 @@ if(WIN32)
 
     WASAPIStream.cpp
     WASAPIStream.h
+  target_include_directories(audiocommon PRIVATE
```

Comment:
Maybe this part should remain with audiocommon or be moved somewhere more generic?

Reply:
I don't know if we have a "more generic" place for this, so I just removed it from that PR and will make it part of WASAPI PR.